src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+.product-card {
+  justify-content: center;
+  padding: 32px 16px 16px;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: $border-radius;
+  box-shadow: 0px 4px 10px rgba(0, 0, 0, 0.3);
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: $card-width - 40px;
+    height: 134px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    color: $main-accent-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    line-height: 14px;
+    margin-bottom: 16px;
+    color: $secondary-color;
+  }
+
+  &__info {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-accent-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    color: $secondary-color;
+
+    &-title {
+      font-size: 12px;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      color: $main-accent-color;
+    }
+  }
+
+  &__buy {
+    display: inline-block;
+    height: 40px;
+    width: 100%;
+
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    text-align: center;
+    color: #fff;
+    padding: 12px 70px;
```

Comment:
The padding `12px 70px` seems to be too specific and might not scale well with different text lengths. Consider using a more flexible approach, such as `padding: 12px 16px` and setting a `min-width` if necessary.